.ci/openshift-ci/buildall_install.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+#
+# Copyright (c) 2020 Red Hat, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Build kata-containers and install into a given destination directory
+#
+
+set -e
+
+if [ -z "$1" ];then
+	echo "Usage: $0 path/to/install/dir"
+	exit 1
+fi
+
+export DESTDIR=$1
+echo "Build and install kata-containers at: ${DESTDIR}"
+
+export PATH=${GOPATH}/bin:$PATH
+
+cidir=$(dirname "$0")/..
+source /etc/os-release || source /usr/lib/os-release
+source "${cidir}/lib.sh"
+
+# The scripts rely on sudo which is not installed in the build environment.
+yum install -y sudo
```

Comment:
- This script assumes an rpm-based system, but there is no check to ensure that.
- I'm assuming all these scripts need to be run as `root`. We try to only run the minimal set of commands as `root` and always with  `sudo`. However, if the enture script needs to be run as `root` (ideally it wouldn't), please add checks to all scripts to ensure they are run as the correct user.

Reply:
This script doesn't need to run as root. However, as it is tied with OpenShift CI, the build environment is actually an OpenShift Build in a CentOS container, with root user and without sudo.